test/suites/depend.bash

```diff
@@ -78,7 +78,7 @@ EOF
     backdate "$BASEDIR3/header.h" "$BASEDIR3/test.c"
     backdate "$BASEDIR4/header.h" "$BASEDIR4/test.c" "$BASEDIR4/header2.h"
 
-    DEPFLAGS="-MD -MF test.d"
+    DEPFLAGS="-g1 -g0 -MD -MF test.d"
```

Comment:
I would prefer a separate test case for this (in the base suite) to make it possible to understand to purpose of the flags.